src/index.tsx

```diff
@@ -12,7 +12,7 @@ import {
 } from './helpers'
 import cssStyles from './styles.css'
 
-type Props = {
+export type EasyCropperProps = {
```

Comment:
As the component is named Cropper, I would name this CropperProps.

Reply:
@ValentinH Can do, but thought due to NPM name it would be best to relate to that i.e `react-easy-crop` therefor 'EasyCropper`

Reply:
Cropper is the name used in the docs so I'd use this :)